src/internal/analytics/selectors.ts

```diff
@@ -1,5 +1,11 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
+import breadcrumbGroupItemStyles from '../../breadcrumb-group/item/styles.css.js';
+import breadcrumbGroupStyles from '../../breadcrumb-group/styles.css.js';
+
+export const getBreadcrumbLinkSelector = (index: number) =>
+  `.${breadcrumbGroupStyles['breadcrumb-group']} .${breadcrumbGroupStyles.item}:nth-child(${index}) .${breadcrumbGroupItemStyles.anchor}`;
```

Comment:
Why do we continue reading this value from DOM? This does not scale.

We should expose a helper in awsuiPlugins API instead

Reply:
As discussed this will come in a follow-up iteration.